src/main/web/src/pages/login.html

```diff
@@ -115,6 +118,7 @@
 	<script type="text/javascript" th:src="@{/controller/static/js/lib/modernizr-custom.js}"></script>
 	<script type="text/javascript" th:src="@{/controller/static/js/lib/jquery.placeholder.js}"></script>
 	<script type="text/javascript" th:src="@{/controller/static/js/lib/select2.min.js}"></script>
+	<script type="text/javascript" th:src="@{/controller/static/js/lib/get-url-parameter.js}"></script>
```

Comment:
Ideally you would put it in the `head`, so that it's available to every scripts or inline script